### PR TITLE
SLLS-14 Compute VM classpath from Java home and pass it to the Java a…

### PR DESCRIPTION
**Pull request by henryju**
…nalyzer

### CODE REVIEW

File: src/test/java/org/sonarsource/sonarlint/ls/mediumtests/JavaMediumTests.java [hunk 1]
```diff
@@ -107,7 +109,40 @@ public void analyzeSimpleJavaFileReuseCachedClasspath() throws Exception {
...
+    String uri = getUri("analyzeSimpleJavaFileOnOpen.java");
+
+    GetJavaConfigResponse javaConfigResponse = new GetJavaConfigResponse();
+    javaConfigResponse.setSourceLevel("1.8");
+    javaConfigResponse.setTest(false);
+    javaConfigResponse.setClasspath(new String[0]);
+    javaConfigResponse.setVmLocation(currentJdkHome.toString());
+    client.javaConfigs.put(uri, javaConfigResponse);
+
+    List<Diagnostic> diagnostics = didOpenAndWaitForDiagnostics(uri, "java", "public class Foo {\n  public static void main() {\n  // System.out.println(\"foo\");\n}\n}");
```
Comment (jblievremont) [inline src/test/java/org/sonarsource/sonarlint/ls/mediumtests/JavaMediumTests.java:128]: Nitpick: I would love to see a test source where the difference in classpath yields in a difference in analysis results. Might be tricky to pull out.

**Review by jblievremont — APPROVED** [review]
LGTM :+1:

**Comment by henryju** [comment]
I found an issue with classpath cache eviction (in fact this is an issue we had even before). This should now make everything work better.

**Review by jblievremont — APPROVED** [review]
LGTM

**Pull request merged**